source/slang/slang-ir-inst-defs.h

```diff
@@ -271,6 +272,12 @@ INST(makeArray, makeArray, 0, 0)
 INST(makeStruct, makeStruct, 0, 0)
 INST(MakeTuple, makeTuple, 0, 0)
 INST(GetTupleElement, getTupleElement, 2, 0)
+INST(MakeResultValue, makeResultValue, 1, 0)
+INST(MakeResultValueVoid, makeResultValueVoid, 0, 0)
```

Comment:
We are very much at the point now where we should go ahead and consistently work it through the IR that *all* functions return a value, and it is just that IR functions that have a `Void` return type return the single global literal of that type, called `void`. Under that model, the `makeResultValueVoid` case is also not needed.